src/main/java/org/example/dao/UserDaoImpl.java

```diff
@@ -10,12 +11,19 @@
 import java.sql.Timestamp;
 import java.util.UUID;
 
-public class UserDaoImpl implements UserDao {
+@Component
```

Comment:
`@Repository`

Reply:
Changed annotation from @Component to @Repository